src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -27,7 +27,8 @@ CORECLR_HOSTING_API(coreclr_initialize,
 
 CORECLR_HOSTING_API(coreclr_shutdown,
             void* hostHandle,
-            unsigned int domainId);
+            unsigned int domainId,
```

Comment:
Right, you cannot modify the ICLRRuntimeHost methods either. This is a COM interface and by COM rules, you need to add a new version of the whole interface, called ICLRRuntimeHost4 (there already is ICLRRuntimeHost3) and put the new method on it.